web/views/auth_view.ex

```diff
@@ -0,0 +1,27 @@
+defmodule CodeCorps.AuthView do
+  use CodeCorps.Web, :view
+
+  def render("show.json", %{token: token, user_id: user_id}) do
+    %{
+      token: token,
+      user_id: user_id,
```

Comment:
Is `user_id` specifically needed separately from the client? If not, we should be able to just decode the JWT on the client and retrieve the users details from that.